tests/Google.Ads.GoogleAds.Tests.csproj

```diff
@@ -1,7 +1,6 @@
 <Project Sdk="Microsoft.NET.Sdk" ToolsVersion="15.0">
   <PropertyGroup>
-    <TargetFrameworks Condition="'$(OS)' == 'Windows_NT'">netcoreapp2.0;net452</TargetFrameworks>
-    <TargetFrameworks Condition="'$(OS)' != 'Windows_NT'">netcoreapp2.0</TargetFrameworks>
+    <TargetFramework>net452</TargetFramework>
```

Comment:
Did you mean to remove Core?

Reply:
Yes. See the description about working around NUnit bug. Specifically, https://github.com/nunit/nunit3-vs-adapter/issues/17